### PR TITLE
Add README and working site map.

### LINKED ISSUE #32: Design site map for browser client
Specify the high-level structure of the application, and how various pages/activities map to URL paths.

### PR DESCRIPTION
**Pull request by ranweiler**
Closes #32.

This site map is meant to be extremely provisional. We expect to change it as the formal specification develops, the overall audit ceremony is finalized, and as we obtain feedback from UX research and user testing.

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### Audit Data Upload
+#### `/audit/upload`
+
+Place for admin users to upload data for audits.
+
+### Audit Seed
+#### `/audit/seed`
+
+View or set the seed fo rthe audit.
```
Comment (dmzimmerman) [inline client/README.md]: "View or set the seed for the audit"

File: client/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Audit repot
```
Comment (dmzimmerman) [inline client/README.md]: "### Audit report"

Comment (dmzimmerman) [inline client/README.md]: Also, in line 36 (which I forgot to flag), "View or set the seed for the audit."

File: client/README.md (hunk 1 above)
Comment (ranweiler) [inline client/README.md]: Amended in fd278e8.

File: client/README.md (hunk 2 above)
Comment (ranweiler) [inline client/README.md]: Amended in fd278e8.

**Comment by ranweiler** [comment]
Thanks for the review @dmzimmerman, I fixed the typos you caught.

**Pull request merged**